src/app/view/components/dot-persona-selector/dot-persona-selector.component.spec.ts

```diff
@@ -166,27 +166,23 @@ describe('DotPersonaSelectorComponent', () => {
     it('should execute "change" event from dot-persona-selector-option', () => {
         hostFixture.whenStable().then(() => {
             spyOn(component.selected, 'emit');
-            spyOn(dropdown.componentInstance, 'toggleOverlayPanel');
```

Comment:
this is been tested on a previous test
https://github.com/dotCMS/core-web/blob/49bd92f77796176a3d1dcf494fa9eed9d9ffb98d/src/app/view/components/dot-persona-selector/dot-persona-selector.component.spec.ts#L143